index.html

```diff
@@ -0,0 +1,462 @@
+      </div>
+    </div>
+  </div>
+
+  <!--achievements-->
+<section id="achievements" style="padding-top:10%;">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-7">
+        <div class="card mb-3" style="min-height:20px; box-shadow:10px 5px 10px #888888">
+          <div class="card-body">
+            <div class="card-text">
+              <h2><b><i>NEWS</i></b></h2>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-7">
+        <div class="card mb-3" style="box-shadow:none;">
+          <div class="card-body">
+            <div class="card-text">
+              <article>
+                <h4><a href="http://technonjr.org/news/techno-india-njr-is-participating-in-the-nvidia-gtc-conference-on-april-12th-2021/" title="Techno India NJR is participating in the NVIDIA GTC conference on April 12th, 2021.">Techno India NJR is participating in the NVIDIA GTC conference on April 12th, 2021.</a></h4>
+                <small>NVIDIA GTC is a global virtual conference and will...</small><hr>
+              </article>
+              <article>
+                <h4><a href="http://technonjr.org/news/placement-2020-21-cognizant-technologies-selects-3-techno-njr-students-at-a-package-of-5-41-lpa/" title="[PLACEMENT 2020-21] Cognizant Technologies Selects 3 Techno NJR students at a Package of 5.41 LPA.">[PLACEMENT 2020-21] Cognizant Technologies Selects 3 Techno NJR students at a Package of 5.41 LPA.</a></h4>
+                <small>[PLACEMENT 2020-21] Cognizant Technologies Selects...</small><hr>
+              </article>
+              <article>
+                <h4><a href="http://technonjr.org/news/thermax-selects-4-techno-njr-students/" title="Thermax selects 4 Techno NJR Students">Thermax selects 4 Techno NJR Students</a></h4>
+                <small>Congratulations to the selected students: 1. Chetan...</small><hr>
+              </article>
+              <article>
+                <h4><a href="http://technonjr.org/news/startup-venture-wricks-by-ex-students-of-techno-india-njr-incubated-at-techno-india-njr-incubation-center-has-won-the-second-prize-of-canadian-12500/" title="Startup Venture Wricks by ex-students of Techno India NJR incubated at Techno India NJR Incubation Center has won the second prize of Canadian $12500/-">Startup Venture Wricks by ex-students of Techno India NJR incubated at Techno India NJR Incubation Center has won the second prize of Canadian $12500/-</a></h4>
+                <small>Startup Venture Wricks by Lokesh Puri Goswami and Kunjpreet...</small>
+              </article>
+            </div>
+          </div>
+        </div>
+      </div>
+  </div>
+  <div class="row">
+    <div class="col-md-5">
+      <a href="http://technonjr.org/news" class="btn btn-lg btn-secondary">READ MORE</a>
+    </div>
```

Comment:
```suggestion
      <a href="" class="btn btn-lg btn-secondary">READ MORE</a>
    </div>
```
Don't use this links of external website. We have to create our own pages later instead.